app/helpers/communicart_mailer_helper.rb

```diff
@@ -19,17 +19,14 @@ def auto_login?(user)
     user.outgoing_delegates.empty?
   end
 
-  def approval_action_url(approval, action = 'approve')
-    opts = {
-      approver_action: action,
-      cart_id: approval.cart_id,
-      version: approval.proposal.version
-
-    }
+  # approve-only at the moment
+  def approval_action_url(approval)
```

Comment:
Would love a different name from this method to distinguish it from the built-in route helpers. Prepending `custom_` feels like cheating, but would do the trick.